dashboard/patient/js/mqtt.js

```diff
@@ -2,51 +2,58 @@ var mqttserver ="";
 var mqttuser ="";
 var mqttpass ="";
 
-    user = sessionStorage.getItem('user');
-    token = sessionStorage.getItem('token');
-    let formData = {
-            user: user,
-            token: token,
-          };
+user = sessionStorage.getItem('user');
+token = sessionStorage.getItem('token');
+let formData = {
+        user: user,
+        token: token,
+    };
 
 (function ($) {
     "use strict";
     
     $.ajax({
         type: "GET",
-        url: "https://healthconnect-server.herokuapp.com/portal/device",
+        url: "https://healthconnect-server.onrender.com/devtkn/portal",
         data : formData,
         crossDomain: true,
         dataType: "json",
         encode: true,
       }).done(function (data) {
-        ////console.log(data.patient[0]._id);
         mqttserver = data.mqttserver;
         mqttuser = data.mqttUser;
         mqttpass = data.mqttPass;
       }).fail(function (data) {
-        window.location.href="../../login/"
+       console.log("Device Portal Check Failed")
       });
 })(jQuery);
 
-var ID = "";
 function startConnect(dev_id) {
-    
-
-    // Generate a random client ID
-
-    // Fetch the hostname/IP address and port number from the form
     host = mqttserver;
     port = 8884;
-    ID = "node-"+dev_id;
+    let ID = "node-"+dev_id;
     
     let act = "Connecting to: " + host + ' on port: ' + port + ', with Node ID - ' + ID;
     // Print output for the user in the messages div
     console.log(act);
     clientID = "clientID-"+dev_id;
     // Initialize new Paho client connection
     client = new Paho.MQTT.Client(host, Number(port), clientID);
+    window.localStorage.setItem(clientID, JSON.stringify(client));
+    
+    // Called when the client connects
+    function onConnect() {
+        // MQTT topic to subscribe 
+        topic = "data/patient/"+sessionStorage.getItem('user')+"/med/"+ID;
+        // Print output for the user in the messages div
+        let act = "Subscribing to: " + topic;
+        console.log(act);
+
+        // Subscribe to the requested topic
```

Comment:
Those ONLINE, unsubscribe them
```suggestion
        server_line = false;
```